src/solvers/refinement/string_constraint_generator.h

```diff
@@ -90,6 +90,18 @@ class array_poolt final
 /// are given as a struct containing a length and pointer to an array.
 array_string_exprt of_argument(array_poolt &array_pool, const exprt &arg);
 
+/// Collection of constraints of different types: existential formulas,
+/// universal formulas, and "not contains" (universal with one alternation).
+struct string_constraintst final
```

Comment:
The name choice is not great as it looks very much like `string_constraintt` and makes it fairly confusing.

Reply:
agreed, any suggestion?

Reply:
`string_constraint_sett`?

Reply:
How painful would it be to keep this `string_constraintst` rename `string_constraintt` to something else?

Reply:
Not very, but if we change we should try to make it a definitive choice.
I guess any choice in the regexp: (`universal_` | `quantified_`)? (`string` | `array`) `_` (`property` | `formula` | `constraint`) `t` could work

Reply:
`string_constraintt` -> `universal_string_constraintt` would make sense to me.